stable/insights-agent/requirements.yaml

```diff
@@ -1,5 +1,5 @@
 dependencies:
   - name: goldilocks
     repository: https://charts.fairwinds.com/stable
-    version: 1.0.0
+    version: 1.1.1
```

Comment:
Seems like at least a minor version bump for the change in Goldilocks to v2 and enabling it by default.